pkgs/by-name/mc/mcap-cli/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+let
+  version = "0.0.38";
+in
+buildGoModule {
+
+  pname = "mcap-cli";
+
+  inherit version;
+
+  src = fetchFromGitHub {
+    repo = "mcap";
+    owner = "foxglove";
+    rev = "releases/mcap-cli/v${version}";
+    hash = "sha256-mwKWf0kJ3uMx1cLUac+AqXgQ1Af3tLDOCTFKgq8FtHE=";
+  };
+
+  vendorHash = "sha256-Gl0zLBTWscKGtVOS6rPRL/r8KHYHpZwoUDbEyCL4Ijk=";
+
+  modRoot = "go/cli/mcap";
+
+  GOWORK="off";
+
+  # copy the local versions of the workspace modules
+  postConfigure = ''
+    chmod -R u+w vendor
+    rm -rf vendor/github.com/foxglove/mcap/go/{mcap,ros}
+    cp -r ../../{mcap,ros} vendor/github.com/foxglove/mcap/go
+  '';
+
+  # tests segfault
+  doCheck = false;
```

Comment:
packages can not be producing segfaults / failing tests without reasons. can you file an bug upstream and resolve the issues? ... looks like a bug was already filed: https://github.com/foxglove/mcap/issues/895
so just disabling the failing test seems fine as there is no network access in the sandbox

Reply:
I added your suggested change and verified the build works.